diffpy/pdfmorph/pdfplot.py

```diff
@@ -55,8 +53,8 @@ def plotPDFs(pairlist, labels=None, offset ='auto', rmin = None, rmax = None):
     if gap == 0:
         plt.legend(loc = 0)
 
-    plt.xlabel("$r (\AA)$")
-    plt.ylabel("$G (\AA^{-1})$")
+    plt.xlabel("$r (\mathrm(\AA))$")
```

Comment:
did you test that this works btw?  I am not sure that my fix actually fixes it.  maybe try it in a jupyter notebook or sthg, or ask one of the guys what the right fix is.....I am not sure.  If you did check that it works, just mention it here.

Reply:
Do we have any 2 dummy PDF files that I can use as input to test?

Reply:
![image](https://user-images.githubusercontent.com/15218640/59367117-7ba2ce00-8d09-11e9-83bb-c86922db5847.png)
yup it should work fine